### ISSUE TITLE
Uknown error, please try again

### ISSUE THREAD

**Issue by Gabirel** [Yufeikang/bcloud#14]
## 系统环境
- Arch Linux
- python3(default)
- pip3(default)

## Screenshot
![image](https://cloud.githubusercontent.com/assets/12933851/26039476/e4476694-394d-11e7-9d38-375c92107c09.png)

## Log

```log
** (bcloud-gui:4102): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-JdCm9EORcT: Connection refused
2017-05-15 09:07:31,346 util.py :23 ERROR Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/bcloud/util.py", line 20, in <module>
    from Crypto.PublicKey import RSA
ModuleNotFoundError: No module named 'Crypto'

2017-05-15 09:07:32,766 App.py :47 DEBUG Traceback (most recent call last):
  File "<frozen importlib._bootstrap>", line 879, in _find_spec
AttributeError: 'DynamicImporter' object has no attribute 'find_spec'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/bcloud/App.py", line 45, in <module>
    from gi.repository import AppIndicator3 as AppIndicator
  File "/usr/lib/python3.6/site-packages/gi/importer.py", line 127, in find_module
    'introspection typelib not found' % namespace)
ImportError: cannot import name AppIndicator3, introspection typelib not found


(bcloud-gui:4102): Gtk-WARNING **: (gtkicontheme.c:2184):gtk_icon_theme_choose_icon: runtime check failed: ((flags & GTK_ICON_LOOKUP_GENERIC_FALLBACK) == 0)
```

## 安装依赖

**其实我已经安装过了**
```log
sudo pip install Crypto
[sudo] password for gabriel: 
Requirement already satisfied: Crypto in /usr/lib/python3.6/site-packages
Requirement already satisfied: shellescape in /usr/lib/python3.6/site-packages (from Crypto)
Requirement already satisfied: Naked in /usr/lib/python3.6/site-packages (from Crypto)
Requirement already satisfied: pyyaml in /usr/lib/python3.6/site-packages (from Naked->Crypto)
Requirement already satisfied: requests in /usr/lib/python3.6/site-packages (from Naked->Crypto)
```

---

可能的类似Issue: https://github.com/Yufeikang/bcloud/issues/5

**Comment by wkcn** [comment]
bcloud里from Crypto.PublicKey import RSA使用的是pycrypto库
pycrypto库和Crypto库是两个不同的库
应该安装pycrypto库~
安装方法：
```
sudo pip install pycrypto
```

**Comment by Gabirel** [comment]
谢谢～解决了
